### PR TITLE
api.did.actor serviceprovider

### PR DESCRIPTION
**Pull request by nissimsan**

### CODE REVIEW

**Comment by nissimsan** [comment]
No oauth needed for this, so there oauth2 element or repo secret

**Comment by brownoxford** [comment]
> No oauth needed for this, so there oauth2 element or repo secret

Is this temporary, or permanent? I understood that we were moving towards making missing/failing OAuth be a test failure condition, but that doesn't have to be the case.

File: packages/tests-postman/data/service-providers.json [hunk 1]
```diff
@@ -56,5 +56,22 @@
...
+                "name": "Transmute DID Actor API",
+                "url": "https://api.did.actor",
+                "contact": {
+                    "name": "Orie Steele",
+                    "email": "orie@transmute.industries"
+                },
+                "markets": [ "steel", "e-commerce" ]
+            },
+            "baseURL": "https://api.did.actor",
+            "vcPrefix": "next",
```
Comment (OR13) [inline packages/tests-postman/data/service-providers.json:73]: I don't think this is right.... probably should be `api`... but I would have to look at the urls and match them up with what is published

Comment (OR13) [inline packages/tests-postman/data/service-providers.json:73]:
```suggestion
            "vcPrefix": "api",
```

Comment (OR13) [reply packages/tests-postman/data/service-providers.json]: Pulled form the console logs... 

```
https://api.did.actor/api/credentials/issue
```

**Review by mkhraisha — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

**Pull request merged**